episodes/chapter6.md

```diff
@@ -512,22 +490,23 @@ When you finish the process, the node configuration file will be updated with th
 
 To make the new API key effective, you need to restart the node by running the command `v6 node stop` and then `v6 node start`.
 
-
 ::::::::::::::::::::::::::::::::::::: challenge
 
-## Challenge 4: Update the API key of a node
+## Challenge 3: Update the API key of a node
 
-1. How do you update the API key of a node? Try to update the API key of the node you just created, without using the `v6 node set-api-key` command.
-2. How do you make sure the new API key is effective?
+1. How to reset the API key of a node via the vantage6 UI? Try to reset the API key of a node you just created.
```

Comment:
I would just say "Reset the API key of the node that you just created in the vantage6 UI"